Formula/parallel.rb

```diff
@@ -1,17 +1,17 @@
 class Parallel < Formula
   desc "Shell command parallelization utility"
   homepage "https://savannah.gnu.org/projects/parallel/"
-  url "https://ftp.gnu.org/gnu/parallel/parallel-20200822.tar.bz2"
-  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200822.tar.bz2"
-  sha256 "9654226a808392c365b1e7b8dea91bf4870bc4f306228d853eb700679e21be09"
-  license "GPL-3.0-or-later"
+  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
+  mirror "https://ftpmirror.gnu.org/parallel/parallel-20200722.tar.bz2"
```

Comment:
Please add a comment that `20200822` is a beta release for Mac (with a link to release notes)

Reply:
@bayandin  hey, thanks for the changes, what do you mean by comment? Where shall it be added?

Reply:
Please add a comment above a line with `url`:
```ruby
  # a comment
  url "https://ftp.gnu.org/gnu/parallel/parallel-20200722.tar.bz2"
```

Reply:
@bayandin done ;)